hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -41,8 +41,8 @@ kcli create network -c 192.168.123.0/24 ocp
 kcli create network -c 192.168.${virtual_router_id}.0/24 --nodhcp -i $cluster_name
 
 cat <<EOF > ./${cluster_name}-plan.yaml
-tag: 4.14.0-rc.1
-ctlplane_memory: 24576
+tag: 4.14.0-rc.6
```

Comment:
Hmm, feels like OCP version should be passed in as env, but not related to this PR...

Reply:
I will do it in a following PR where I am also going to improve the scripts to support more nodes